introduction.md

```diff
@@ -1 +1,4 @@
-introduction for the repository dojo_rules of the course mastering github at code school. i like the ruby path
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?